test/Controller/DigestControllerTest.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+namespace test\eLife\Journal\Controller;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use test\eLife\Journal\Providers;
+
+/**
+ * @backupGlobals enabled
+ */
+final class DigestControllerTest extends PageTestCase
+{
+    use Providers;
+
+    /**
+     * @before
+     */
+    public function enableFeatureFlag()
+    {
+        $_ENV['FEATURE_DIGEST_CHANNEL'] = true;
+    }
```

Comment:
needs an `@after` too to reset the environment variable to the default

Reply:
`@backupGlobals enabled` handles it.